backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java

```diff
@@ -48,14 +50,13 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
       CameraController cameraController = ServerController.getInstance().getCameraController();
       Camera camera = cameraController.getCameraById(Integer.parseInt(camID));
       String tags = request.getParameter("tags");
-      List<String> tagList = null;
-      
+
+      List<String> tagList = new ArrayList<String>();
```

Comment:
That is because you assign tagList with a static type of List<String> to a new ArrayList. You could also do List<A> taglist = new ArrayList<B> where B is a subclass of A. Why is tagList stored as a list and not an ArrayList?